pkg/test/client.go

```diff
@@ -13,26 +13,34 @@ import (
 // NewFakeClient creates a fake K8s client with ability to override specific Get/List/Create/Update/StatusUpdate/Delete functions
 func NewFakeClient(t *testing.T, initObjs ...runtime.Object) *FakeClient {
 	client := fake.NewFakeClientWithScheme(scheme.Scheme, initObjs...)
-	return &FakeClient{client, t, nil, nil, nil, nil, nil, nil}
+	return &FakeClient{client, t, nil, nil, nil, nil, nil, nil, nil, nil, nil}
```

Comment:
I'm curious - we do we need to specify all the nil values when nil is the default value?
```suggestion
	return &FakeClient{Client: client, T: t}
```

Reply:
Fixed in https://github.com/codeready-toolchain/toolchain-common/pull/48/commits/306f81888e72953684d26874f19edca862f54f94